### DISCUSSION TITLE
A question about .skip

### DISCUSSION THREAD

**Discussion by Northxw** [cypress-io/cypress#16910] [Questions and Help] [answered]
Hello, Cypress users，I have a cypress test script as shown below：

```js
describe('Cypyress Test Hooks', function () {

    before(function () {
        cy.log('i am in before')
    })

    after(function () {
        cy.log('i am in after')
    })

    beforeEach(function () {
        cy.log('i am in beforeEach')
    })

    afterEach(function () {
        cy.log('i am in afterEach')
    })

    it('Cypress Test Case 1', function () {
        cy.log('i am in Test Case 1')
    })

    specify.skip('Cypress Test Case 2', function () {
        cy.log('i am in Test Case 2')
    })
})
```
When I execute the test script through Cypress TestRunner, I found that the after hook has no effect, I don’t know why. The execution result is shown in the figure：

![image](https://user-images.githubusercontent.com/38708989/121798841-3ba9b500-cc5b-11eb-910b-9f21eaa90d67.png)

Environment: 
```text
Cypress package version: 7.5.0
Cypress binary version: 7.5.0
Electron version: 12.0.0-beta.14
Bundled Node version:
14.15.1
```

**Comment by jennifer-shehane** [comment, marked as answer]
The `after` hook is run, it just does not display in the Cypress UI since the after hook is always shown in the last test (which shows no UI in this example since it was skipped). 

This is this issue: https://github.com/cypress-io/cypress/issues/1846

**Reply by Northxw** [reply]
Thank you very much.
